io/file/binary writer/a92a6e2b-166e-4b43-8dc4-3674d8ef5faf.json

```diff
@@ -4,21 +4,25 @@
     "inputs": {
       "2f960cd2-ce95-47dd-9dfa-7b89eefe48e3": {
         "type": "string",
-        "name": "path"
+        "name": "path",
+        "description": "Receives the path of the generated binary file.\n\nExample:\n\"c:/image.png\"@1"
```

Comment:
I would append this small notice for all `path` parameters as we already added it to all path related nodes.
"(To keep the application portable use "/" as path separator.)"
Example: io/file/Path resolver

Reply:
Done